### PR TITLE
[#145] Add support for messageproperties to support ordered loading through special _order message code

### PR DESCRIPTION
**Pull request by mseaton**

### CODE REVIEW

**Review by Ruhanga — APPROVED** [review]
Looks good @mseaton, thanks for this intuitive solution. Where as the motivation might have been that of _"overriding properties"_ , this might also come in handy for _"message property interpolation"_.

File: readme/messageproperties.md [hunk 1]
```diff
@@ -9,5 +9,11 @@ messageproperties/
 There can be as many message properties internationalization files as needed.
 This domain differs from most others since nothing from its configuration is persisted in database, everything is stored in the runtime memory upon starting the Initializer.
 
+If one needs to configure the order in which message properties files load, for example if one message properties
+file contains message codes that should override those in another in the same locale, one can do this by utilizing a 
+special message code named "_order".  Two message properties files in the same locale will be loaded in order based
+first on the value of any "_order" property defined, and second based on the alphabetical order of the 
+absolute path of the file.
```
Comment (mks-d) [inline readme/messageproperties.md:16]: @mseaton could you document what happens if `_order` is specified in only one of two files for the same locale?

I suspect that one will be loaded first?

**Review by mks-d — COMMENTED** [review]
I would kindly request a commit headline à la
>[#145] messageproperties to support ordering through special _order property key.

... or whatever you deem appropriate.

File: readme/messageproperties.md (hunk 1 above)
Comment (mseaton) [reply readme/messageproperties.md]: Done.  Let me know if that's clear.

**Pull request merged**